recipe/meta.yaml

```diff
@@ -15,8 +15,11 @@ build:
   number: 0
   script:
     - export DISABLE_NUMCODECS_AVX2=""  # [unix]
+    - export DISABLE_NUMCODECS_SSE2=""  # [arm64]
     - set DISABLE_NUMCODECS_AVX2=""  # [win]
     - {{ PYTHON }} -m pip install . -vv
+  # numcodecs is not currently supported on s390x
+  skip: true  # [s390x]
```

Comment:
Skip py<36 too, see https://github.com/zarr-developers/numcodecs/blob/fee2beec148ddebf8022f4e8ed33804bee31c06f/setup.py#L348

Reply:
Additionally, does this package really not build correctly on s390x?